packages/@sanity/cli/src/commands/blueprints/initBlueprintsCommand.ts

```diff
@@ -1,95 +1,77 @@
-import {join} from 'node:path'
-import {cwd} from 'node:process'
-
 import {type CliCommandDefinition} from '../../types'
 
 const helpText = `
-Examples
-  # Create a new Blueprint manifest file
+Arguments
+  <dir>  Path to initialize the Blueprint in
+
+Options
+  --blueprint-type, --type <json|ts|js>    Type of Blueprint to create
```

Comment:
Should we mention the default?

Reply:
I suppose right now we should just say "json" because the prompt won't let you select ts or js. You can still use the flag, but support for dynamic blueprint files isn't great yet...